tests/test_timer/timer_utils.py

```diff
@@ -0,0 +1,39 @@
+from tests.test_timer import Timer
+
+__all__ = ["log_time", "get_singleton_manager"]
+
+
+def get_singleton_manager(enable_logging: bool = True):
+    """
+    Return the Timer. If not has not yet been initialized, initialize and
+    return. If it has, return the existing Timer.
+    """
+    if Timer._instance is None:
+        Timer._instance = Timer(enable_logging=enable_logging)
+    return Timer._instance
+
+
+def log_time(func):
+    """
+    Decorator to time functions. Times for the function are stored using
+    the class and function names.
+    """
+
+    def wrapper(*args, **kwargs):
+        if args and isinstance(args[0], object):
```

Comment:
I think everything is an object? so the isinstance check is a no-op

```python
>>> isinstance(1, object)
True
```

Reply:
Yeah I’m gonna clean this up. Realized there’s some leftover code as well with the func_name